android/app/src/main/java/com/now/naaga/data/remote/retrofit/service/LetterService.kt

```diff
@@ -0,0 +1,35 @@
+package com.now.naaga.data.remote.retrofit.service
+
+import com.now.naaga.data.remote.dto.ClosedLetterDto
+import com.now.naaga.data.remote.dto.OpenLetterDto
+import com.now.naaga.data.remote.dto.post.PostLetterDto
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface LetterService {
+    @POST("/letters")
+    suspend fun registerLetter(
+        @Body postLetterDto: PostLetterDto,
+    ): Response<OpenLetterDto>
+
+    @GET("/letters/nearby")
+    suspend fun getNearbyLetters(
+        @Query("latitude") latitude: String,
+        @Query("longitude") longitude: String,
```

Comment:
```suggestion
        @Query("latitude") latitude: Double,
        @Query("longitude") longitude: Double,
```
노션을 보니 해당 쿼리 타입이 Double인 것 같아요!
아니면 String으로 해도 되는건가요?! 몰라서 물어봅니다!

Reply:
반영했습니다!